nautilus_trader/persistence/catalog/parquet.py

```diff
@@ -208,12 +208,14 @@ def write_chunk(
             self._fast_write(table=table, path=path, fs=self.fs)
         else:
```

Comment:
My thinking is that it works without the partitioning flag here, perhaps an other check can be used here to enter the write_dataset code block.